db/schema.rb

```diff
@@ -73,6 +75,15 @@
     t.index ["user_id"], name: "index_user_roles_on_user_id"
   end
 
+  create_table "stores", force: :cascade do |t|
+    t.string "name"
+    t.string "status"
```

Comment:
Did we want to make a default status of pending?

Reply:
Also on this, do we want a string or should we do enums for the statuses of pending, active and suspended?